src/Restorer/AbstractRestorer.php

```diff
@@ -34,9 +37,10 @@ public function checkBinary(): string
     {
         $process = new Process([$this->binary, '--version']);
         $process->run();
+
         if (!$process->isSuccessful()) {
-            throw new \InvalidArgumentException(\sprintf(
-                "Error while trying to process '%s', check configuration for binary '%s",
+            throw new \LogicException(\sprintf(
```

Comment:
I would use a `\RuntimeException` as well here. `\LogicException` kind of means that the user did something wrong, that may not be the case here.

Reply:
I hesitated. Ok to change that.